onnxruntime/python/_pybind_state.py

```diff
@@ -40,9 +40,9 @@
         cudnn_bin_dir = os.path.join(cudnn_path, "bin")
 
         if not os.path.isfile(os.path.join(cudnn_bin_dir, f"cudnn64_{version_info.cudnn_version}.dll")):
-            raise ImportError(f"cuDNN {version_info.cudnn_version} not installed in {cudnn_bin_dir}. "
-                              f"Set the CUDNN_HOME environment variable to the path of the 'cuda' directory "
-                              f"in your CUDNN installation if necessary.")
+            print(f"cuDNN {version_info.cudnn_version} not installed in {cudnn_bin_dir}. "
+                  f"Set the CUDNN_HOME environment variable to the path of the 'cuda' directory "
```

Comment:
What is the user expected to do with this print statement if cudnn is obtained from the PATH?

Reply:
I don't think this will alone fix https://github.com/microsoft/onnxruntime/issues/7965. For python vers >=3.8, PATH won't be consulted to find the deps.

Reply:
The user is doing something non recommended, I don't think we can help them in that case.